milight_ibox2/milight_ibox2_control.py

```diff
@@ -214,17 +267,17 @@ def send_command(self, light_command):
             self.ibox_seq &= 0xFF
 
             self.socket_send(cmd)
-            data = self.socket_recv()
-            if data:
-                if len(data) != 8:
+            rx_data, (rx_addr, rx_port) = self.socket_recv()
```

Comment:
This unpacking will fail if None is received

Reply:
I agree, and I've pushed a fix to this branch.